README.md

```diff
@@ -4,35 +4,40 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 
 ## Commands
 
-- `odf set`:
-  - `recovery-profile <profile>`: Set the recovery profile to favor new IO, recovery, or balanced mode with options `high_client_ops`, `high_recovery_ops`, or `balanced`. The default is `balanced`.
-  - `full <ratio>`: Update the ceph osd full ratio after which Ceph automatically prevents any I/O operations on OSDs.
-  - `nearfull <ratio>`: Update the ceph osd nearfull ratio in case Ceph returns the nearfull osds message when the cluster reaches the capacity specified.
-  - `backfillfull <ratio>`:  Update the ceph osd backfillfull ratio in case ceph will deny backfilling to the OSD that reached the capacity specified.
-  - `ceph`
-    - `log-level <daemon> <subsystem> <log-level>`: Set the log level for Ceph daemons like OSD, mon, mds etc. More information about the ceph subsystems can be found [here](https://docs.ceph.com/en/latest/rados/troubleshooting/log-and-debug/#ceph-subsystems)
-- `odf get`:
-  - `recovery-profile`: Get the recovery profile value.
-  - `health`: Check health of the cluster and common configuration issues.
-  - `dr-health [ceph status args]`: Print the ceph status of a peer cluster in a mirroring-enabled environment thereby validating connectivity between ceph clusters. Ceph status args can be optionally passed, such as to change the log level: --debug-ms 1.
-  - `dr-prereq <PeerManagedClusterName>` : Print the status of pre-requisites for Disaster Recovery between peer clusters.
-  - `mon-endpoints`: Print mon endpoints.
-- `odf purge-osd <ID>`: Permanently remove an OSD from the cluster.
-- `odf maintenance`: [Perform maintenance operations](docs/maintenance.md) on mons or OSDs. The mon or OSD deployment will be scaled down and replaced temporarily by a maintenance deployment.
-  - `start <deployment-name>`
-    - `[--alternate-image <alternate-image>]` : (optional) Start a maintenance deployment with an optional alternative ceph container image
-  - `stop <deployment-name>`: Stop the maintenance deployment and restore the mon or OSD deployment
-- `odf subvolume`:
-  - `ls`: Display all the subvolumes
-  - `delete <subvolume> <filesystem> <subvolumegroup>`: Deletes the stale subvolumes
-- `odf operator`:
-  - `rook`:
-    - `set`: Set the property in the rook-ceph-operator-config configmap.
-    - `restart` : Restart the Rook-Ceph operator
-- `odf restore`:
-  - `mon-quorum`: Restore the mon quorum based on a single healthy mon since quorum was lost with the other mons
-  - `deleted`: Restore the ceph resources which are stuck in deleting state due to underlying resources being present in the cluster
-- `odf help` : Display help text
+- `odf`:
+  - `ceph <args>` : Run a Ceph CLI command. Supports any arguments the ceph command supports. See [Ceph](https://docs.ceph.com/en/pacific/start/intro/) docs for more.
```

Comment:
Let's add these new commands as the last commands in the list. They are advanced commands, so I'd rather the other commands be first in the doc. Also, to keep the diff simple how about not changing the structure of the other commands in this PR. The old structure looks ok to me, but feel free to open a separate PR for that if desired.